gguf-py/gguf/constants.py

```diff
@@ -1242,6 +1244,20 @@ class MODEL_TENSOR(IntEnum):
         MODEL_TENSOR.FFN_DOWN,
         MODEL_TENSOR.FFN_UP,
     ],
+    MODEL_ARCH.GRANITE_MOE: [
+        MODEL_TENSOR.TOKEN_EMBD,
+        MODEL_TENSOR.OUTPUT_NORM,
```

Comment:
```suggestion
        MODEL_TENSOR.OUTPUT_NORM,
        MODEL_TENSOR.OUTPUT,
```
(adding the line for `MODEL_TENSOR.OUTPUT`)
Otherwise I was getting `ValueError: Can not map tensor 'lm_head.weight'` when converting <https://huggingface.co/ibm/PowerMoE-3b>